pages/understanding-json-schema/reference/enum.md

```diff
@@ -9,50 +9,96 @@ next:
   url: /understanding-json-schema/reference/const
 ---
 
-The `enum` [keyword](../../learn/glossary#keyword) is used to restrict a value to a fixed set of values.
-It must be an array with at least one element, where each element is
-unique.
+The `enum` keyword in JSON Schema allows you to restrict a value to a set of predefined options.
+It requires an array with at least one unique element, and any value validated against this schema must exactly match one of the specified values.
```

Comment:
"one unique element" is imprecise.  `enum` requires at least one element, and all elements must be unique. 

"One unique element" reads as "can have whatever, but at least one of the many elements must be unique", which isn't the same thing.